app/over_react_redux/todo_client/lib/src/components/user_selector.dart

```diff
@@ -0,0 +1,99 @@
+library todo_client.src.components.user_selector;
+
+import 'package:collection/collection.dart';
+import 'package:over_react/over_react.dart';
+import 'package:over_react/over_react_redux.dart';
+
+import 'package:todo_client/src/store.dart';
+import 'package:todo_client/src/models/user.dart';
+import 'package:todo_client/src/components/shared/avatar_with_colors.dart';
+import 'package:todo_client/src/components/shared/material_ui.dart';
+import 'package:todo_client/src/components/shared/menu_overlay.dart';
+
+part 'user_selector_trigger.dart';
+// ignore: uri_has_not_been_generated
+part 'user_selector.over_react.g.dart';
+
+UiFactory<UserSelectorProps> ConnectedUserSelector = connect<AppState, UserSelectorProps>(
+    mapStateToPropsWithOwnProps: (state, ownProps) {
+      return (UserSelector()
+        ..selectedUser = ownProps.selectedUserId.isNotEmpty
+            ? state.users.firstWhere((user) => user.id == ownProps.selectedUserId)
+            : null
+        ..users = state.users
+      );
+    },
+)(UserSelector);
+
+@Factory()
+UiFactory<UserSelectorProps> UserSelector =
+    // ignore: undefined_identifier
+    _$UserSelector;
+
+@Props()
+class _$UserSelectorProps extends UiProps {
+  String selectedUserId;
+  User selectedUser;
+  List<User> users;
+  Function(String userId) onUserSelect;
```

Comment:
Using the same logic as my other required prop comments, this could also be annotated as one.